particles/Restaurants/Reservations.recipes

```diff
@@ -48,12 +39,15 @@ particle ReservationMultiAnnotation in 'source/ReservationAnnotation.js'
   consume annotation
 
 recipe MakeReservations
-  // &calendarReserve
   use as restaurants
-  use #reservation as event
   use #selected as restaurant
-  create as descriptions
+  create #reservation as event
+  create as calendar_descriptions
+  create as annotation_descriptions
```

Comment:
odd question: I thought we're following camelCase, I understand parser is happy with either but should we at least agree internally?

Reply:
I don't know why I did it that way, I'll change it.